lib/utils/import.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+const assert = require('assert');
+const { isIdentifier, isImportDeclaration, isMemberExpression } = require('../utils/types');
+
+module.exports = {
+  getSourceModuleNameForIdentifier,
+  getSourceModuleName,
+};
+
+/**
+ * Gets the name of the module that an identifier was imported from,
+ * if it was imported
+ *
+ * @param {Object} context the context of the ESLint rule
+ * @param {node} node the Idenfifier to find an import for
+ * @returns {string | undefined} The name of the module the identifier was imported from, if it was imported
+ */
+function getSourceModuleNameForIdentifier(context, node) {
+  const [program] = context.getAncestors(node);
+  const importDeclaration = program.body
+    .filter(isImportDeclaration)
+    .find(importDeclaration =>
+      importDeclaration.specifiers.some(
+        specifier => specifier.local.name === getSourceModuleName(node)
+      )
+    );
+
+  return importDeclaration ? importDeclaration.source.value : undefined;
+}
+
+function getSourceModuleName(node) {
```

Comment:
This doesn't actually have anything to do with imports, does it? My understanding is that it gets the name of whatever the top-most member in a change of member expressions is.

Reply:
That's a good point. Although it's only used by `getSourceModuleNameForIdentifier`. I'm open to moving it back if you'd like. `getSourceModuleNameForIdentifier` also needs to be renamed to make it clear that it gets the import path.